plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
When you say "using the IP range notation even for single IP addresses" do you mean something like:

```
123.45.12.234:123.45.12.234
```

Personally, I see no problem in mixing ranges and single addresses in one form field as long as the exact notation is documented clearly, like:

```
142.58.0.0:142.58.79.255
142.58.88.0:142.58.101.26
123.45.12.234
```

But, I'd be happy to discuss further or to open a ticket to get wider input.